### ISSUE TITLE
[CocoaPods] EarlGrey dependency error can return

### ISSUE THREAD

**Issue by willlarche** [material-components/material-components-ios#1175]
### Overview
Error when installing pods can return even after the dev gets EarlGrey working properly.

```[!] Unable to satisfy the following requirements:

- `EarlGrey` required by `Podfile`
- `EarlGrey (= 1.6.2)` required by `Podfile.lock`

None of your spec sources contain a spec satisfying the dependencies: `EarlGrey, EarlGrey (= 1.6.2)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.
```

Running `pod repo update` and then trying again works to make it go away but the error is scary and long.

### Reproduction steps
I believe this happened when EarlyGrey updated their repo somehow.

### Proposed solutions
1. Put `pod repo update` in our prep_all script.
1. Add that fix to the documentation / FAQ

**Comment by ajsecord** [comment]
Solution 1 is not good because we'd be forcing the user to update their pods—they might have good reason to not update to new versions. I agree the message is overly scary...let's go with solution 2.

**Comment by willlarche** [comment]
Fixed in #1176 - Went with solution 1
